whats_fresh/whats_fresh_api/templates/vendor.html

```diff
@@ -112,7 +147,132 @@ <h3 class="title">Products</h3>
 </div>
 {% endblock content %}
 {% block footer %}
+    <script src="https://code.jquery.com/jquery-1.10.2.min.js"></script>
+    <script src="http://rawgit.com/Logicify/jquery-locationpicker-plugin/master/dist/locationpicker.jquery.js"></script>
 <script>
+
+// var checkboxStatus = 1;
+$('#us3-address').hide();
+$('#us3-address-new').hide();
+
+
+    var newStreet = $('meta[name=dbstreet]').attr('value');
+    var newCity = $('meta[name=dbcity]').attr('value');
+    var newState = $('meta[name=dbstate]').attr('value');
+    var newZip = $('meta[name=dbzip]').attr('value');
+
+    var dbAddress = newStreet + "," + newCity + "," + newState + "," + newZip;
+
+{% if vendor_form.instance.pk != None  %}
```

Comment:
Try expressing this as ```{% if vendor_form.instance.pk %}```. It should work, unless my assumptions about the Django rendering system are wrong.